modules/integration/tests/src/test/java/org/wso2/appmanager/integration/test/cases/publisher/webapp/lifecycle/management/ChangeStateFromCreateToInReviewTestCase.java

```diff
@@ -0,0 +1,153 @@
+    public void testChangeStateFromCreateToInReviewWithInValidUsers(String userName, String password, String uuid)
+            throws Exception {
+        APPMPublisherRestClient publisherRestClient = new APPMPublisherRestClient(backEndUrl);
+        // Login to publisher by invalid user.
+        publisherRestClient.login(userName, password);
+        HttpResponse httpResponse = publisherRestClient.changeState(uuid, AppmTestConstants.LifeCycleStatus
+                .SUBMIT_FOR_REVIEW);
+        JSONObject responseData = new JSONObject(httpResponse.getData());
+        // Logout from publisher by invalid user.
+        publisherRestClient.logout();
+        int responseCode = httpResponse.getResponseCode();
+        assertTrue(responseCode == 401, "Excepted status code is 401 for user :" + userName + ". But received " +
+                "status code is " + responseCode);
+        assertEquals(responseData.getString(AppmTestConstants.STATUS), "Access Denied", "Changing web app life cycle " +
+                "state from create to review allowed for user : " + userName + " who has insufficient privileges to " +
+                "change life cycle status.");
+    }
+
+    @AfterClass(alwaysRun = true)
+    public void closeDown() throws Exception {
+        // Deleted created web app by AppCreator.
+        appmPublisherRestClient.deleteApp(app1Uuid);
+        appmPublisherRestClient.deleteApp(app2Uuid);
+        appmPublisherRestClient.deleteApp(app3Uuid);
+        // Logout from publisher by AppCreator user.
+        appmPublisherRestClient.logout();
+    }
+
+    private String createWebApp(String appPrefix) throws Exception {
+        PolicyGroup defaultPolicyGroup = WebAppUtil.createDefaultPolicy();
+        HttpResponse response = appmPublisherRestClient.addPolicyGroup(defaultPolicyGroup);
+        String policyId = WebAppUtil.getPolicyId(response);
+        List<WebAppResource> webAppResources = WebAppUtil.createDefaultResources(policyId);
+        WebApp webApp = WebAppUtil.createBasicWebApp(appCreatorUserName, appName + appPrefix, context +appPrefix,
+                                                     appVersion, "http://wso2.com/", webAppResources);
+        appmPublisherRestClient.createWebApp(webApp);
+        return webApp.getAppId();
+    }
+
+    @DataProvider
+    public Object[][] validUserModeDataProvider() throws Exception {
+        User adminUser = appMServer.getSuperTenant().getTenantAdmin();
+        User appCreator = appMServer.getSuperTenant().getTenantUser("AppCreator");
+        return new Object[][]{
+                new Object[]{adminUser.getUserName(), adminUser.getPassword(), app1Uuid},
+                new Object[]{appCreator.getUserName(), appCreator.getPassword(), app2Uuid}
+        };
+    }
+
+    @DataProvider
+    public Object[][] inValidUserModeDataProvider() throws Exception {
+        User appPublisher = appMServer.getSuperTenant().getTenantUser("AppPublisher");
```

Comment:
Fixed in https://github.com/wso2/product-app-manager/pull/357/commits/d5adfb28b2122712d0da0911c0d4921cf5bd98d9